Src/VsVimShared/VsVimHost.cs

```diff
@@ -771,6 +771,11 @@ public override bool ShouldCreateVimBuffer(ITextView textView)
                 return false;
             }
 
+            if (!_vsAdapter.IsTextEditorView(textView))

```

Comment:
Changing this particular function always makes me a bit nervous. VS is such a diverse ecosystem and there are always components out there which find new and odd ways to create windows out that surprise me. In the past I've ended up disabling VsVim for scenarios that I wasn't aware of. 

The good news about changing this though is if we get it wrong people complain ... fast. And while impactful to users it can also be fixed rather fast. 

The advice I was given by the editor team when I started VsVim was to focus on the `TextViewRoles` of an `ITextView` for controlling where VsVim should be displayed. If a particular window doesn't want VsVim then it shouldn't use the structured role, etc ... That advice works up to a point but there are just so many real examples where it falls down (like this one).

Reply:
Gulp! Fingers crossed.